tide-log/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+authors = [
+    "Tide Developers"
+]
+description = "Tide middleware for logging"
+documentation = "https://docs.rs/tide"
+keywords = ["tide", "http", "web", "framework", "async", "middleware"]
```

Comment:
Crates.io is limited to 5 keywords here, I believe. You should remove one of these and perhaps replace another with "logging", as I think that describes this middleware crate better.

My personal suggestion is:

```toml
keywords = ["tide", "web", "async", "middleware", "logging"]
categories = ["network-programming", "logging", "asynchronous"]
```

Similar to https://github.com/rustasync/tide/blob/master/tide-compression/Cargo.toml

Reply:
Thanks @fairingrey! This pasted your suggestion directly for `keywords`, but categories, I used 
```
categories = [
  "logging",
  "network-programming",
  "web-programming::http-server",
]
```
instead, since while the middleware is inherently async, the logging particularly isn't! And I think the web bit is probably more relevant? :)